### ISSUE TITLE
PDFUA-223: Figure with visible text, but only with ActualText | Fail

### ISSUE THREAD

**Issue by zakkinsey** [zakkinsey/pdfa-test-7#223]
Jira issue originally created by user @MarkusErle:

This example demonstrates the failure condition that applies for an image that conveys non-text information, but [alt text](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#alt-text) is missing. [ActualText](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#actualtext) may additionally be set, if the image contains visible text. [Alt text](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#alt-text) is more appropriate than just [ActualText](https://www.pdfa.org/glossary-of-accessibility-terminology-in-pdf/#actualtext), because the image contains non-text information.

**Comment by zakkinsey** [comment]
@MarkusErle added a link to #222: This issue clones #222

**Comment by zakkinsey** [comment]
Comment created by @PaulRayius:

Left off here in the meeting - debate over required use of Alt or not.

**Comment by zakkinsey** [comment]
@zakkinsey unassigned issue from @MarkusErle as part of jira-&gt;github migration
